github3/repos/repo.py

```diff
@@ -1653,6 +1653,17 @@ def release(self, id):
             json = self._json(self._get(url), 200)
         return self._instance_or_null(Release, json)
 
+    def release_by_tag_name(self, tag_name):
+        """Get a relase by tag name
```

Comment:
Release not Relase ;)

Also please add a `.` to the end since this should be a sentence.